### ISSUE TITLE
Rainbow Effect

### ISSUE THREAD

**Issue by KaiH-0** [wherrera10/asus-lighting#1]
This project is amazing! Is there a way to make it rainbow? I have tried but failed.

Thanks for taking the time to read this :)

**Comment by wherrera10** [comment]
No, I do not know how to do this (I do not think that my machine had that
feature).


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
